CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing Guidelines
```

Comment:
Adapted this from the CLI's: https://github.com/CircleCI-Public/circleci-cli/blob/b3d26e795c6df1f94c4a4ab9a9e5e2a0815d8cc4/CONTRIBUTING.md